sleuth_backend/solr_module/solr_module.py

```diff
@@ -0,0 +1,11 @@
+import pysolr
+
+solr_url = 'http://localhost:8983/solr/'
```

Comment:
Instead of duplicating this URL here, do something like

```python
from sleuth_backend.settings import HAYSTACK_CONNECTIONS
solr_url = HAYSTACK_CONNECTIONS['default']['URL']
```